audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/exception/ApplicationAsyncExceptionHandler.java

```diff
@@ -33,7 +35,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(ApplicationAsyncExceptionHandler.class);
 
     @Override
-    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object... objects) {
+    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object[] objects) {
```

Comment:
Not sure this is right.  The method parameters declares that the array of Objects can be nullable, but the first thing the method does is check to make sure that the array is not null.  Also the method assumes that the other objects are not null and are not tested.  Third,  is that we stopped using the Object test for null for the Optional test.

Reply:
I will look into a better way of doing this. The `requireNonNull` was added to make sure the Steam had values.

Reply:
Updated

Reply:
Updated